### PR TITLE
Update adding-private-network.md

### PR DESCRIPTION
**Pull request by jerry-li-dev**

### CODE REVIEW

File: src/pages/docs/collaborating-in-postman/adding-private-network.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ The default view is your team's [Private API Network](https://go.postman.co/netw
 
 In the modal, choose the workspace containing the API you want to add to the network. You can select one or more APIs to add to the network at one time. For each API selected, specify the versions to add. Select **Next** and then the folder to add the APIs to. Select **Add to Network**.
 
-<img alt="Add API Modal" src="https://assets.postman.com/postman-docs/add-new-api-to-network-v8.jpg" width="500px"/>
+<img alt="Add API Modal" src="https://assets.postman.com/postman-docs/add-to-team-private-api-network.jpg" width="500px"/>
```
Comment (jkonrath-postman) [inline src/pages/docs/collaborating-in-postman/adding-private-network.md:64]: Throw a border on this one. (It looks like it has one, but not on the bottom side?)

File: src/pages/docs/collaborating-in-postman/adding-private-network.md [hunk 2]
```diff
@@ -89,7 +89,7 @@ You can create new folders from the Private API Network view. Click __Add New__,
 
 Once you've created the folder, click __Add New__ and select __APIs__ to add APIs to your Team's API Network. You can choose the workspace and then select as many APIs you want to add to the network at one time. For each API selected, specify the versions to add. Click __Add__.
 
-<img alt="Add APIs Workspace" src="https://assets.postman.com/postman-docs/add-apis-workspace.jpg" width="400px"/>
+<img alt="Add APIs Workspace" src="https://assets.postman.com/postman-docs/add-to-team-private-api-network.jpg" width="400px"/>
```
Comment (jkonrath-postman) [inline src/pages/docs/collaborating-in-postman/adding-private-network.md:92]: Same border thing. (Same image for both, though.)

**Review by jkonrath-postman — CHANGES_REQUESTED** [review]

File: src/pages/docs/collaborating-in-postman/adding-private-network.md [hunk 3]
```diff
@@ -61,7 +61,7 @@ The default view is your team's [Private API Network](https://go.postman.co/netw
 
 In the modal, choose the workspace containing the API you want to add to the network. You can select one or more APIs to add to the network at one time. For each API selected, specify the versions to add. Select **Next** and then the folder to add the APIs to. Select **Add to Network**.
 
-<img alt="Add API Modal" src="https://assets.postman.com/postman-docs/add-new-api-to-network-v8.jpg" width="500px"/>
+<img alt="Add API Modal" src="https://assets.postman.com/postman-docs/add-to-team-private-api-network-2.jpg" width="500px"/>
```
Comment (jkonrath-postman) [inline src/pages/docs/collaborating-in-postman/adding-private-network.md:64]: Is something up with the S3 hosting on these? I'm getting an access denied message when I try to open the URL.

**Review by jkonrath-postman — CHANGES_REQUESTED** [review]
Look into the image problems here - is it an S3 issue?

File: src/pages/docs/collaborating-in-postman/adding-private-network.md (hunk 3 above)
Comment (jerry-li-dev) [reply src/pages/docs/collaborating-in-postman/adding-private-network.md:64]: Ok, I just fixed the issue. I think the AWS upload errored out and it didn't show an error.

**Review by jkonrath-postman — APPROVED** [review]
LGTM

**Pull request merged**